### PR TITLE
add whatsapp contact button

### PR DESCRIPTION
**Pull request by cryptosalomao**
This PR addresses #44

### CODE REVIEW

File: app/constants/support.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const WHATSAPP_CONTACT_NUMBER = '+5599985049935'
```
Comment (nicolasburtey) [inline app/constants/support.ts:1]: should be +503 6983 5117

File: app/screens/settings-screen/settings-screen.tsx [hunk 2]
```diff
@@ -1,25 +1,36 @@
...
+
 
 const styles = EStyleSheet.create({
   screenStyle: {
     backgroundColor: palette.lighterGrey
   },
 })
 
-export const SettingsScreen = ({navigation}) => {
+export const SettingsScreen: any = ({ navigation }) => {
```
Comment (nicolasburtey) [inline app/screens/settings-screen/settings-screen.tsx:33]: we're trying to move forward a strongly type database. could you add the correct type here?

File: app/constants/support.ts (hunk 1 above)
Comment (cryptosalomao) [reply app/constants/support.ts:1]: Oh yeah, of course. I just added mine as an example

File: app/screens/settings-screen/settings-screen.tsx (hunk 2 above)
Comment (cryptosalomao) [reply app/screens/settings-screen/settings-screen.tsx:33]: Sure.

File: app/constants/support.ts (hunk 1 above)
Comment (nicolasburtey) [reply app/constants/support.ts:1]: > Oh yeah, of course. I just added mine as an example
you'll risk getting a lot of spam ;)

File: app/screens/settings-screen/settings-screen.tsx [hunk 3]
```diff
@@ -32,7 +32,11 @@ const styles = EStyleSheet.create({
   },
 })
 
-export const SettingsScreen: ScreenType = ({ navigation }) => {
+type Props = {
+  navigation: any,
```
Comment (nicolasburtey) [inline app/screens/settings-screen/settings-screen.tsx:36]: we're just pushing the `any` one depth further here :) but that'll do it for now. thanks.

**Comment by nicolasburtey** [comment]
merged. didn't pay attention but we use yarn not npm. so I deleted package-json.lock in main

**Pull request merged**